cli/dcoscli/marathon/main.py

```diff
@@ -293,14 +294,31 @@ def _about():
 
 def _get_resource(resource):
     """
-    :param resource: optional filename for the application or group resource
+    :param resource: optional filename or http(s) url
+    for the application or group resource
     :type resource: str
     :returns: resource
     :rtype: dict
     """
     if resource is not None:
-        with util.open_file(resource) as resource_file:
-            return util.load_json(resource_file)
+        if os.path.isfile(resource):
+            with util.open_file(resource) as resource_file:
+                return util.load_json(resource_file)
+        else:
+            try:
+                http.silence_requests_warnings()
```

Comment:
Already answered this: https://github.com/mesosphere/dcos-cli/pull/255#discussion_r34038661